fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -503,6 +505,22 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun disableAppBarDragging() {
+        if (binding.homeAppBar.layoutParams != null) {
+            val appBarLayoutParams = binding.homeAppBar.layoutParams as CoordinatorLayout.LayoutParams
+            val appBarBehavior = AppBarLayout.Behavior()
+            appBarBehavior.setDragCallback(
+                object : AppBarLayout.Behavior.DragCallback() {
+                    override fun canDrag(appBarLayout: AppBarLayout): Boolean {
+                        return false
+                    }
+                },
+            )
+            appBarLayoutParams.behavior = appBarBehavior
+        }
+        binding.homeAppBar.setExpanded(true)
```

Comment:
I see this change being added in the last commit.
Is this proposed as a fix for the issue seen in tests - described more here - https://github.com/mozilla-mobile/firefox-android/pull/852#issuecomment-1453784159 ?

`binding.homeAppBar.setExpanded(true)` would regress another behavior as described here - https://github.com/mozilla-mobile/firefox-android/pull/852#discussion_r1118488158

If the patch cannot be improved to avoid both issues we could check with UX to temporary approve one since the original issue seems a bit more important to fix.

Let me know what your thoughts are.

Reply:
Hello @Mugurell,
This fix is workaround with less code. If we need proper implementation, then for simple fix a lot more code would be added. As you mentioned on [#852 (Comment)](https://github.com/mozilla-mobile/firefox-android/pull/852#discussion_r1118488158) that make sure to add least changes.